docs/basics/functions.md

```diff
@@ -832,13 +832,14 @@ Which would return the following:
 }]
 ```
 
-### split_array
+### split_array (leaf_array)
```

Comment:
Does leaf_array need to be there in brackets?

Reply:
So user knows there is a leaf_array option with split_array.